relay-server/src/services/project.rs

```diff
@@ -610,9 +612,19 @@ impl Project {
             })
             .collect();
 
-        if !denied_buckets.is_empty() {
-            let scoping = self.scoping();
-            if let Some(scoping) = scoping {
+        if let Some(scoping) = self.scoping() {
+            if !disabled_namespace_buckets.is_empty() {
+                utils::reject_metrics(
```

Comment:
Here i was torn between allowing multiple outcomes to be supported in `reject_metrics` but I feel like this implementation is a bit more explicit even though it's more verbose.